code/game/verbs/who.dm

```diff
@@ -1,7 +1,7 @@
 
 /client/verb/who()
 	set name = "Who"
-	set category = "OOC"
+	set category = null
```

Comment:
Explain?

Reply:
Yeah, no reason to remove it. I use it pretty regularily to keep a number of how many people we have on.